processing/src/main/java/org/apache/carbondata/processing/csvreaderstep/CsvInputMeta.java

```diff
@@ -162,6 +168,8 @@ private void readData(Node stepnode) throws KettleXMLException {
       blocksID = XMLHandler.getTagValue(stepnode, "blocksID");
       partitionID = XMLHandler.getTagValue(stepnode, "partitionID");
       escapeCharacter = XMLHandler.getTagValue(stepnode, "escapeCharacter");
+      quoteCharacter = XMLHandler.getTagValue(stepnode, "quoteCharacter");
+      commentCharacter = XMLHandler.getTagValue(stepnode, "commentCharacter");
```

Comment:
please keep the same sequence:  escapeCharacter -> quoteCharacter -> commentCharacter